actors/builtin/miner/miner_test.go

```diff
@@ -899,53 +1057,509 @@ func TestCommitments(t *testing.T) {
 			require.NoError(t, st.SaveDeadlines(rt.AdtStore(), deadlines))
 			// Phew!
 
-			rt.ReplaceState(st)
-			rt.ExpectAbort(exitcode.ErrForbidden, func() {
-				actor.preCommitSector(rt, &params, preCommitConf{})
-			})
-			rt.ReplaceState(&prevState)
-			rt.Reset()
+			rt.ReplaceState(st)
+			rt.ExpectAbort(exitcode.ErrForbidden, func() {
+				actor.preCommitSector(rt, &params, preCommitConf{})
+			})
+			rt.ReplaceState(&prevState)
+			rt.Reset()
+		}
+
+		{ // Target must not be terminated
+			params := *upgradeParams
+			st := getState(rt)
+			prevState := *st
+			quant := st.QuantSpecForDeadline(dlIdx)
+			deadlines, err := st.LoadDeadlines(rt.AdtStore())
+			require.NoError(t, err)
```

Comment:
The harness has helpers like `getDeadlines`, `getDeadline` to avoid all this noise.

Edit: I see you just moved this. Minimising the moves would help make the real delta here clear.